operators-examples/qiskit-dev-operator/operator/build/Dockerfile

```diff
@@ -1,13 +1,11 @@
 FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
 
-RUN echo "Qiskit-Dev-Operator by Emerging Tech Red Hat"
-
-ENV OPERATOR=/usr/local/bin/qiskit-operator \
+ENV OPERATOR=/usr/local/bin/qiskit-dev-operator \
     USER_UID=1001 \
-    USER_NAME=qiskit-operator
+    USER_NAME=qiskit-dev-operator
 
 # install operator binary
-COPY build/_output/bin/operator ${OPERATOR}
+COPY build/_output/bin/qiskit-dev-operator ${OPERATOR}
```

Comment:
It may be common to hard-code the source directory. But, thinking out loud, since the target dir is a variable perhaps the source dir should too?